torch/csrc/distributed/c10d/TCPStoreLibUvBackend.cpp

```diff
@@ -1326,11 +1330,11 @@ bool LibUVStoreDaemon::waitKeys(
 }
 
 int64_t LibUVStoreDaemon::size() {
```

Comment:
Nit: would it be better to change the return type here to size_t? Or does that break things?

Reply:
can be size_t, but keep it to avoid checking more code.